lecture_2/01_channels/example3/main.go

```diff
@@ -29,7 +29,7 @@ func main() {
 		ch <- i
 	}
 
-	// close(ch)
```

Comment:
Yes, this probably shouldn't be in a PR since this isn't the part of HW, it happened during the lecture and it accidentally stayed with all changes.